microros_extensions/zephyr_serial_transport.c

```diff
@@ -17,6 +17,24 @@
 u8_t uart_in_buffer[RING_BUF_SIZE];
 u8_t uart_out_buffer[RING_BUF_SIZE];
 
+struct ring_buf out_ringbuf, in_ringbuf;
+
+static void uart_fifo_callback(struct device *dev){ 
+    while (uart_irq_update(dev) && uart_irq_is_pending(dev)) {
+        if (uart_irq_rx_ready(dev)) {
```

Comment:
What happens when `uart_irq_rx_ready` return false?

Reply:
It means the IRQ callback has been called for other reason different from rx ready event. This should not happen since only `uart_irq_rx_enable(platform->uart_dev);` has been enabled.

Reply:
So, in case it happens what should we do? Should we `break` or `return`?

Reply:
Done